src/content/plugins/watch-ignore-plugin.mdx

```diff
@@ -10,9 +10,26 @@ contributors:
 Ignore the specified files, i.e. those matching the provided paths or regular expressions, while in [watch mode](/configuration/watch).
 
 ```javascript
-new webpack.WatchIgnorePlugin({ paths });
+new webpack.WatchIgnorePlugin(paths);
```

Comment:
This doesn't seem right to me as per https://github.com/webpack/webpack/blob/68c4a2a0a9a9c12c55e2ba2a5273dd26958e4027/lib/WatchIgnorePlugin.js#L112:

<img width="584" alt="CleanShot 2022-01-08 at 22 27 46@2x" src="https://user-images.githubusercontent.com/1091472/148647890-09019a2e-c25d-45b3-8665-33fbea6b7fb1.png">